.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,45 @@
+name: "CodeQL"
+
+on:
+  # TODO(clason): remove when done testing
+  pull_request:
+    branches: [ "master" ]
+  schedule:
+    - cron: '42 0 * * *'
+
```

Add a `workflow_dispatch`, it's nice to have on scheduled workflows for testing purposes.

```suggestion
  workflow_dispatch:
```